### ISSUE TITLE
Gradle plugin v1.5.0 seems to be broken

### ISSUE THREAD

**Issue by markusthoemmes** [alenkacz/gradle-scalafmt#16]
Just tried to install the plugin via the gradle `plugins` syntax (as marketed on their plugins website):

```groovy
plugins {
    id "cz.alenkacz.gradle.scalafmt" version "1.5.0"
}
```

Version 1.5.0 doesn't seem resolvable though. 1.4.0 works like a charm. 1.5.0 is installable via the instructions in README.md. Is gradle "native" support abandoned?

**Comment by alenkacz** [comment]
To test this, I added the plugin to one of my projects and the build seems to be passing https://github.com/alenkacz/postgres-scala/commit/d7351c6961a062991c60b40b4c14ab4ca68eb699

Are you resolving against jcenter? Can you share some error message or whole build.gradle so that I can help?

Thanks

**Comment by markusthoemmes** [comment]
Sure thing, sorry for taking so long.

The usual plugin applying way as lined out in your readme works like a charm. I was looking into gradle's short-notation way of doing it and referred to:

https://plugins.gradle.org/plugin/cz.alenkacz.gradle.scalafmt

Where the newest version doesn't seem to be pushed to per:

```
> Could not resolve all files for configuration ':common:scala:classpath'.
   > Could not find gradle.plugin.cz.alenkacz:gradle-scalafmt:1.5.0.
     Searched in the following locations:
         https://plugins.gradle.org/m2/gradle/plugin/cz/alenkacz/gradle-scalafmt/1.5.0/gradle-scalafmt-1.5.0.pom
         https://plugins.gradle.org/m2/gradle/plugin/cz/alenkacz/gradle-scalafmt/1.5.0/gradle-scalafmt-1.5.0.jar
```

I assume it's just a matter of publishing the artifacts to the correct server?

**Comment by timyates** [comment]
Hiya, Tim here from Gradle.

There was a [bug in versions prior to 0.9.7 of the `plugin-publish-plugin`](https://discuss.gradle.org/t/plugin-authors-please-use-the-latest-plugin-publish-plugin-others-may-result-in-a-broken-upload/23573), where artifacts would silently not be pushed into the repo.

This unfortunately has affected v`1.5.0` of the `gradle-scalafmt` plugin.

I see you have [updated the plugin to a newer version](https://github.com/alenkacz/gradle-scalafmt/commit/1a6e5dd0413afb0893e2db2da965c0bf04f70564#diff-c197962302397baf3a4cc36463dce5eaR10), so it should just be a case of pushing a new version (`1.5.1`?) to the plugin portal

Once this is done, I can removed the broken `1.5.0` version if you'd like?

Thanks (and sorry about that).

Let us know here [or in the forums](https://discuss.gradle.org/c/help-discuss/plugin-portal) if you need more assistance.

**Comment by alenkacz** [comment]
oh @timyates thank you! Just released 1.5.1 - feel free to remove 1.5.0
